--- ./poppler-cpp.pc.cmake	2023-09-27 14:03:45.576478352 +0200
+++ ./poppler-cpp.pc.cmake	2023-09-27 13:57:58.476472656 +0200
@@ -1,6 +1,6 @@
-prefix=@LIBRARY_LIB@
-libdir=@LIBRARY_LIB@/lib
-includedir=@LIBRARY_LIB@/include
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
 
 Name: poppler-cpp
 Description: cpp backend for Poppler PDF rendering library
